mmpose/models/detectors/bottom_up.py

```diff
@@ -222,12 +224,13 @@ def forward_test(self, img, img_metas, return_heatmap=False, **kwargs):
                 outputs, outputs_flip, self.test_cfg['num_joints'],
                 self.test_cfg['with_heatmaps'], self.test_cfg['with_ae'],
                 self.test_cfg['tag_per_joint'], img_metas['flip_index'],
-                self.test_cfg['project2image'], base_size)
+                self.test_cfg['project2image'], base_size,
+                use_udp=self.use_udp)
```

Comment:
And, we can use `self.test_cfg['use_udp']`  here.

Reply:
done